packages/nightwatch-api/src/index.ts

```diff
@@ -6,26 +6,88 @@ import section from 'nightwatch/lib/page-object/section';
 
 let nightwatchClient: Api;
 
-export { stopWebDriver, closeSession } from './client';
+/**
+ * This variable represents the Nightwatch webdriver client.
+ * This is the main part fo this package.
```

Comment:
Looks like a typo ;) "part of"

Reply:
Wow! 🤣 Thank you so much! Adding you to contributors list. Can you check the whole docs site after this has been merged?

Reply:
It will be my pleasure. :)